cron.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euo pipefail # See http://redsymbol.net/articles/unofficial-bash-strict-mode/
+IFS=$'\n\t'
+
+# Execute via crontab by sol@quaoar1:
+# 00 9 * * * cd /home/sol/git/mabxml-elasticsearch ; bash cron.sh >> logs/cron.sh.log 2>&1
+
```

Comment:
You mean this exact entry? Or an entry for mabxml-elasticsearch in general? The current entry is for `mabxml-elasticsearch-staging`. I will add the production entry during production deployment.